src/components/tabs/_tabs.scss

```diff
@@ -33,7 +34,8 @@
     display: flex;
     align-items: center;
     justify-content: space-between;
-    padding: rem(13px) rem(16px);
+    padding: 0 $spacing-md;
+    height: rem(40px);
```

Comment:
Just wanted to double-check if the `padding`/`height` changes here are intentional.

Reply:
Yes, adding the height centers it vertically so no need for top/bottom padding.